manifests/base/paddock/deployment.yaml

```diff
@@ -1,32 +1,18 @@
-apiVersion: apps.openshift.io/v1
-kind: DeploymentConfig
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   labels:
-    app: paddock
     app.kubernetes.io/component: paddock
   name: paddock
 spec:
   replicas: 1
   revisionHistoryLimit: 2
   selector:
-    app: paddock
-    app.kubernetes.io/component: paddock
-  triggers:
-    - type: ConfigChange
```

Comment:
how does a rollout work in the future? once GH built a new image, isn't the ImageStream responsible for pulling new images and deploying them?

Reply:
https://docs.openshift.com/container-platform/4.13/openshift_images/triggering-updates-on-imagestream-changes.html might be another alternative